autoreject/utils.py

```diff
@@ -461,3 +460,17 @@ def _fast_map_meg_channels(info, pick_from, pick_to,
     mne.set_log_level(verbose)
 
     return fmd['data']
+
+
+def _my_setup_dots(mode, info, coils, ch):
+    """
+    Monkey patcher for MNE's _setup_dots that changed parametrization
+    in version 0.24
+    """
+    from mne.forward._field_interpolation import _setup_dots
+    from setuptools.version import pkg_resources
+    if pkg_resources.parse_version(mne.__version__) < \
```

Comment:
another alternative is to use this: https://github.com/mne-tools/mne-python/blob/f1e6097c36e7232514d67d4b04af568e36ac5758/mne/utils/check.py#L66. Probably more robust?

Reply:
thanks for the pointer! I have used this now